lib/gollum-lib/filter/tags.rb

```diff
@@ -98,10 +109,9 @@ def process_tag(tag)
   #   if it is not.
   #
   def process_include_tag(tag)
-    return unless /^include:/.match(tag)
+    return html_error('Cannot process include directive: no page name given') if tag.length <= 8
     page_name          = tag[8..-1]
-    resolved_page_name = ::File.expand_path(page_name, "/"+@markup.dir)
-
+    resolved_page_name = ::File.expand_path(page_name, "#{::File::SEPARATOR}#{@markup.dir}")
```

Comment:
Right. If we want to rebuild the API for searching/retrieving objects from the repo to use `Pathname`, we should also do it here.